src/cards/ha-weather-card.html

```diff
@@ -35,8 +32,6 @@
 (function () {
```

Comment:
@balloob Should this wrapper go away now that we aren't using it to keep variables in scope, or does 'use strict' still provide value?